doc/ref_arch/kubernetes/chapters/chapter04.md

```diff
@@ -115,7 +116,7 @@ Architecture they must be implemented as per the following specifications:
 |`ra2.ntw.009`|Multiple connection points with multiplexer/meta-plugin|When a multiplexer/meta-plugin is used, the additional non-default connection points must be managed by a CNI-conformant Network Plugin.|[req.gen.ost.01](chapter02.md#23-kubernetes-architecture-requirements)|
 |`ra2.ntw.010`|User plane networking|When hosting workloads matching the Network Intensive profile, CNI network plugins that support the use of DPDK, VPP, or SR-IOV must be deployed as part of the networking solution.|[infra.net.acc.cfg.001](chapter02.md#223-cloud-infrastructure-software-profile-requirements)|[All are available under `example_net_attach_defs:`](../../../ref_impl/cntt-ri2/chapters/chapter04.md#431-installation-on-bare-metal-infratructure)|
 |`ra2.ntw.011`|NATless connectivity|When hosting workloads that require source and destination IP addresses to be preserved in the traffic headers, a CNI plugin that exposes the pod IP directly to the external networks (e.g. Calico, MACVLAN or IPVLAN CNI plugins) is required.|[req.inf.ntw.14](chapter02.md#23-kubernetes-architecture-requirements)|
-|`ra2.ntw.012`|Optional Device Plugins|When hosting workloads matching the Network Intensive profile that require the use of FPGA or other Acceleration Hardware, a Device Plugin for that FPGA or Acceleration Hardware may be used.|[e.cap.016](chapter02.md#221-cloud-infrastructure-software-profile-capabilities)|
+|`ra2.ntw.012`|Device Plugins|When hosting workloads matching the Network Intensive profile that require the use of FPGA or other Acceleration Hardware, a Device Plugin for that FPGA or Acceleration Hardware must be used.|[e.cap.016](chapter02.md#221-cloud-infrastructure-software-profile-capabilities)|
```

Comment:
Add SR-IOV also.

Reply:
If adding SR-IOV, [e.cap.016](https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/kubernetes/chapters/chapter02.md#221-cloud-infrastructure-software-profile-capabilities) ib Ch 02 should also be modified.

Reply:
Should the newly added ra2.app.009 be modified?

Reply:
>  If adding SR-IOV, e.cap.016 ib Ch 02 should also be modified.

SR-IOV is covered in `e.cap.013`

Reply:
> Should the newly added ra2.app.009 be modified?

Yes,

Reply:
SR-IOV added to `ra2.ntw.012` and  `ra2.app.009` in f42f5cd.

Reply:
> 
> 
> > If adding SR-IOV, e.cap.016 ib Ch 02 should also be modified.
> 
> SR-IOV is covered in `e.cap.013`

Yes, but since the Requirement Trace only refers to e.cap.016 either e.cap.016 should be modified or e.cap.013 should be added to the Requirement Trace.

Reply:
I've added e.cap.013 to the requirement trace in 2a5a062